app/commands/git/sync_pull_requests_for_repo.rb

```diff
@@ -0,0 +1,174 @@
+module Git
+  class SyncPullRequestsForRepo
+    include Mandate
+
+    def initialize(repo, created_after: nil)
+      @repo = repo
+      @created_after = created_after
+    end
+
+    def call
+      fetch!
+      create!
+    end
+
+    private
+    attr_reader :pull_requests, :repo, :created_after
+
+    def create!
+      pull_requests.each do |pr|
+        ::Git::PullRequest.new(
+          node_id: pr[:pr_id],
+          number: pr[:pr_number],
+          author_github_username: pr[:author],
+          repo: pr[:repo],
+          event: pr,
+          reviews: pr[:reviews].map do |review|
+            ::Git::PullRequestReview.new(
+              node_id: review[:node_id],
+              reviewer_github_username: review[:user][:login]
+            )
+          end
+        ).save!
+      rescue ActiveRecord::RecordNotUnique
+        nil
+      end
+    end
```

Comment:
Could you move `create!` below `fetch!` as it sits below it in the `call` method pls?

Reply:
Alternatively you rename `fetch!` to `pull_requests`  and memoize it.